Makefile

```diff
@@ -734,7 +734,11 @@ OPTIMIZE    = -O0
 LTO_FLAGS   = $(OPTIMIZE)
 else
 OPTIMIZE    = -Os
+ifeq ($(TARGET),$(filter $(TARGET),$(F1_TARGETS)))
```

Comment:
I will certainly come back to it, but as it is now totally blocking for further progress it needs to be disabled. Will raise an issue so it doesnt get forgotten.

Debugging was quite nasty as the problem was occuring during the DMA execution where no CPU is involved. Hard to track the exact event where its triggered. On top of that it was not occuring with all timers and DMA channels.